js/plot/util/drawbolus.js

```diff
@@ -390,6 +390,8 @@ module.exports = function(pool, opts) {
       html: function(group, d) {
         var bolus = pluckBolus(d);
         var justBolus = (bolus.normal === commonbolus.getMaxValue(d)) && !d.carbInput;
+        var isAnimasCombo = bolus.annotations && Array.isArray(bolus.annotations) &&
+          bolus.annotations.length > 0 && bolus.annotations[0].code === 'animas/bolus/extended-equal-split';
```

Comment:
This isn't going to work if the annotation isn't the first. In my data I know for a fact there are some where there's also an annotation re: unknown time of bolus cancellation. More robust method for searching for the annotation please?